### PR TITLE
Add Toast and Rating

### PR DESCRIPTION
**Pull request by viru2001**
## Added Rating and Toast component

rating toast

[Deploy preview](https://deploy-preview-13--cool-ui.netlify.app/pages/docs.html#rating)

### CODE REVIEW

File: components/Toast/toast.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const closeBtns = document.querySelectorAll(".btn-toast-close");
+const successDemoBtn = document.querySelector("#success-toast-demo-btn");
+const successToast = document.querySelector("#toast-success");
+const errorDemoBtn = document.querySelector("#error-toast-demo-btn");
+const errorToast = document.querySelector("#toast-error");
+
+// close toast when clicked on close icon button
+closeBtns.forEach(btn => {
+  btn.addEventListener("click", () => {
+    btn.parentElement.parentElement.style.display = "none";
```
Comment (bishnoimukesh) [inline components/Toast/toast.js:10]: why are you using the parentElement two times?

**Review by bishnoimukesh — APPROVED** [review]
look good to go

File: components/Toast/toast.js (hunk 1 above)
Comment (viru2001) [reply components/Toast/toast.js:10]: beacuase this is my html structure 

```html
 <div
      id="toast-success"
      class="toast text-sm d-flex align-center p-3 rounded-sm d-none"
    >
      Added Successfully !
      <button type="button" class="btn btn-icon">
        <i class="fas fa-times text-md btn-toast-close ml-4"></i>
      </button>
   </div>

```

File: components/Toast/toast.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    </button>
+
+    <!-- error toast -->
+    <div
+      id="toast-error"
+      class="toast text-sm d-flex align-center p-3 rounded-sm d-none"
+    >
+      Try Again Later !
+      <button type="button" class="btn btn-icon">
+        <i class="fas fa-times text-md btn-toast-close ml-4"></i>
```
Comment (irajbhinde) [inline components/Toast/toast.html:36]: Nothing major but for good UI you can decrease the cross size here, cross is looking a little too big

**Review by irajbhinde — COMMENTED** [review]
Looks good to me. Alternative to using a dark color for the star was taking a dark color star font from font awesome itself but your approach was appropriate as well.

File: components/Toast/toast.html (hunk 2 above)
Comment (viru2001) [reply components/Toast/toast.html:36]: I think it should be big , otherwise user will have problem while using that button

**Pull request merged**